en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
-**ALTITUDE: Altitude Control**
+#### Altitude Mode
 
 Altitude Mode is the safest non GPS guided mode appropriate for beginners learning how to fly. In this mode the altitude is controlled by the PX4 autopilot. The throttle will determine your airspeed, at 50% throttle the aircraft will hold its current altitude with a preset cruise air speed. You can descend and climb with pitch up and down command. In Altitude Mode you have full angle control over Pitch, Roll and Yaw. Once you center all the sticks you will have a level flying aircraft at the current altitude. If the wind blows the aircraft will drift in the direction of the wind.
 
 ![FW Altitude Mode](../../images/flight_modes/altitude_FW.png)
  
- 
-![GPS](../../images/flight_modes/GPS_s.png) **POSITION: Position Control**
+
+#### Position Mode
+
+![GPS](../../images/flight_modes/GPS_s.png)
 
 This is the GPS guided mode. You can fly in Position Mode if you need precision in flight, keeping straight line course, and still maintaining the control of the aircraft with your RC. The altitude is controlled by the PX4 autopilot. The throttle will determine your airspeed, at 50% throttle the aircraft will hold its current altitude with a preset cruise speed. You can descend and climb with pitch up and down command. Position Mode is almost like Altitude Mode with full angle control over Pitch, Roll and Yaw. But Once you center all the sticks you will have a level flying aircraft at the current altitude and GPS guided straight line course. If the wind blows the aircraft will keep its direction of flight and course and will fight the wind.
 
 ![FW Position Mode](../../images/flight_modes/position_FW.png)
  
  
-## Autonomous Modes
+### Autonomous Modes
 
 Autonomous or Auto modes do not require any input from the pilot.
- 
-**PAUSE: Auto Pause**
-
-You can use Pause in case of emergency or during a mission in order to Hold the aircraft at its current position autonomously, while you get time to access the situation either emergency or mission related. When you activate Pause mode with a pre-programmed switch the airplane will start circling around at fixed GPS position, holding its current altitude during the circles. During hold you are not able to control the craft with the RC. It is the first line of defense if you loose control over the aircraft.
- 
- 
-![GPS](../../images/flight_modes/GPS_s.png) **RETURN: Return to Launch (RTL) / Return to Home**
 
-The aircraft returns to the home position and hovers in a circle above the home position at a preset altitude.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **MISSION**
+#### Pause (Auto Pause)
```

Comment:
Needs to be renamed to "Hold Mode"

Reply:
No it is Auto Pause. and in px4 1.6.5 + qgc 3.2.5 it was called Pause...

Reply:
So using recent daily PX4 firmware and QGC 3.3 I see these options. No pause .... ?
![image](https://user-images.githubusercontent.com/5368500/37583129-1fcb301c-2ba4-11e8-9cb5-f654d30bba02.png)